packages/preset-uno/src/rules/behaviors.ts

```diff
@@ -2,6 +2,83 @@ import { Rule } from '@unocss/core'
 import { handler as h } from '../utils'
 import { colorResolver } from './color'
 
+const outlineStyle = [
+  'none',
+  'auto',
+  'dotted',
+  'dashed',
+  'solid',
+  'double',
+  'groove',
+  'ridge',
+  'inset',
+  'outset',
+  'inherit',
+  'initial',
+  'revert',
+  'unset',
+]
+
+const parseOutlineSize = (s: string) => {
+  const propName = ['width', 'offset'].find(item => s.startsWith(item)) || 'width'
+  const size = h.bracket.fraction.rem((s.replace(/^(offset\-|width\-)/, '')))
+  if (size) {
+    return {
+      [`outline-${propName}`]: size,
+    }
+  }
+}
+
+export const outline: Rule[] = [
+  ['outline', { 'outline-style': 'solid' }],
+  [
+    /^outline-(.+)$/, (match, config) => {
+      const [, d] = match
+
+      const handlerList = [
```

Comment:
Why handlers are needed? Could we use `if ... else` directly?

Reply:
I have updated it

Reply:
EMM, because I don't think these blocks are coupled, when I find logic, I need to read it from top to bottom, rather than directly find the corresponding branch with the problem. 

However, there is not much logic in this function, which will also cause the problem of seemingly over design. Then listen to your opinion!

It seems that our biological clock is the opposite. I will deal with the mail immediately when I see it. I'm very sorry that I didn't repair it according to your opinion.

Reply:
No worry about the timezone at all, I was just a bit rushed to push the new release :P